### PR TITLE
Central chapter for thermostats and integrators

### LINKED ISSUE #3913: Central chapter for thermostats and integrators
The number of thermostats and integrators has increased in 4.2, and their documentation has improved substantially. They are currently documented in chapters "3. Setting up the system" and "5. Running the simulation", respectively. However, they are tightly coupled to each other. I think the user guide would be more intuitive if thermostats were moved into chapter "5. Running the simulation".

The restructured user guide would look like this:

3. Setting up the system
   1. Setting global variables in Python
   2. Cellsystems
   3. CUDA
4. Setting up particles
5. Running the simulation
   1. Thermostats
      * Langevin
      * LB
      * DPD
      * Isotropic
      * Brownian
      * Stokesian
   2. Integrators
      * Velocity Verlet
      * Isotropic NPT
      * Steepest descent
      * Stokesian Dynamics
   3. Rotational degrees of freedom and particle anisotropy

### PR DESCRIPTION
**Pull request by jngrad**
Fixes #3913

Description of changes:
- move thermostats and integrators to the same chapter
- mention the Brownian integrator (links to the Brownian thermostat)

### CODE REVIEW

File: doc/sphinx/running.rst [hunk 1]
```diff
@@ -381,3 +403,314 @@ The near field (so-called lubrication) correction is planned. For now,
...
+If the feature ``ROTATION`` is compiled in, the rotational degrees of freedom are
+also coupled to the thermostat. If only the first two arguments are
+specified then the friction coefficient for the rotation is set to the
+same value as that for the translation.
+A separate rotational friction coefficient can be set by inputting
+``gamma_rotate``. The two options allow one to switch the translational and rotational
+thermalization on or off separately, maintaining the frictional behavior. This
+can be useful, for instance, in high Péclet number active matter systems, where
+one only wants to thermalize only the rotational degrees of freedom and
+translational motion is affected by the self-propulsion.
```
Comment (reinaual) [inline doc/sphinx/running.rst]:
```suggestion
If the feature ``ROTATION`` is compiled in, the rotational degrees of freedom are
also coupled to the thermostat. If only the first two arguments are
specified then the friction coefficient for the rotation is set to the
same value as that for the translation.
A separate rotational friction coefficient can be set by inputting
``gamma_rotate``. The two options allow one to switch the translational and rotational
thermalization on or off separately, maintaining the frictional behavior. This
can be useful, for instance, in high Péclet number active matter systems, where
one only wants to thermalize the rotational degrees of freedom and
translational motion is affected by the self-propulsion.
```

File: doc/sphinx/running.rst [hunk 2]
```diff
@@ -381,3 +403,314 @@ The near field (so-called lubrication) correction is planned. For now,
...
+corresponds to a diffusion within the Wiener process:
+
+.. math:: \sigma_p^2 = 2 \frac{kT}{\gamma} \cdot \Delta t
+
+Each velocity component random walk variance :math:`\sigma_v^2` is defined by the heat
+component:
+
+.. math:: \sigma_v^2 = \frac{kT}{m}
+
+Note that the velocity random walk is propagated from zero at each step.
```
Comment (reinaual) [inline doc/sphinx/running.rst]:
```suggestion
Note: the velocity random walk is propagated from zero at each step.
```
To match the style of the note below

**Review by reinaual — DISMISSED** [review]
I just saw two things in the moved text

File: doc/sphinx/running.rst (hunk 1 above)
Comment (jngrad) [reply doc/sphinx/running.rst]: thanks!

File: doc/sphinx/running.rst (hunk 2 above)
Comment (jngrad) [reply doc/sphinx/running.rst]: done

**Review by reinaual — APPROVED** [review]

**Pull request merged**